src/wmtk/TriMesh.cpp

```diff
@@ -275,22 +293,25 @@ Tuple TriMesh::face_tuple_from_id(long id) const
 
     );
     assert(is_ccw(f_tuple));
-    assert(is_valid(f_tuple));
+    assert(is_valid_slow(f_tuple));
     return f_tuple;
 }
 
-bool TriMesh::is_valid(const Tuple& tuple) const
+bool TriMesh::is_valid(const Tuple& tuple, ConstAccessor<long>& hash_accessor) const
 {
     int offset = tuple.m_local_vid * 3 + tuple.m_local_eid;
-    return tuple.m_local_vid >= 0 && tuple.m_local_eid >= 0 && tuple.m_global_cid >= 0 &&
-           autogen::auto_2d_table_ccw[offset][0] >= 0;
-}
+    const bool is_connectivity_valid = tuple.m_local_vid >= 0 && tuple.m_local_eid >= 0 &&
```

Comment:
your formmatter isn't consistent in how it separates out these multi-line things (netiher was the code before tbh)
'id rather see either all statements in a mult-line form or all in a single line

Reply:
Looks like we need to update the clang format file then. I honestly don't care about the formatting here as long as it is consistent across all machines.